packages/graphiql-plugin-code-exporter/src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+// / <reference types="vite/client" />
```

Comment:
We had an issue with out eslint config which always broke triple-slash directives (should by fixed by now)
```suggestion
/// <reference types="vite/client" />
```

Reply:
Not fixed for me :/

![image](https://user-images.githubusercontent.com/16143594/195809225-e198b1ec-e74b-4c67-8cb5-cbbcc36ff523.png)

Reply:
It's also like that in the other packages so I don't think this was fixed